src/components/v2/V2Create/StartOverButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Modal } from 'antd'
+import { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+
+import { editingV2ProjectActions } from 'redux/slices/editingV2Project'
+
+export default function StartOverButton() {
+  const [loading, setLoading] = useState<boolean>(false)
+  const [startOverModalVisible, setStartOverModalVisible] =
+    useState<boolean>(false)
+
+  const history = useHistory()
+
+  const resetV2CreateFlow = () => {
+    setLoading(true)
+    editingV2ProjectActions.resetState()
+    // TODO: push search query, check for it in each for and reset? or will redux+reload cover it?
```

Comment:
Is this TODO still needed?

Reply:
No lol my bad I actually accidentally requested review before this was completely ready